proposals/wgs/wg-lifecycle.md

```diff
@@ -0,0 +1,92 @@
+## SUMMARY:
+
+This document covers everything you need to know about the creation and retirement (“lifecycle”) of a working group within Kubeflow. General project governance information can be found in the [governance doc].
+Out of scope for this document: [subproject] creation.
+
+[Creation]
+[Retirement]
+
+## [Creation]
+### Prerequisites for a WG
+- [ ] Read [wg-governance.md]
+- [ ] Ensure all WG Chairs, Technical Leads, and other leadership roles are [community members]
+- [ ] Send an email to the Steering Committee <steering@kubeflow.org> to scope the WG and get provisional approval.
+- [ ] Look at the checklist below for processes and tips that you will need to do while this is going on. It's best to collect this information upfront so you have a smoother process to launch
+- [ ] Follow the [WG charter process] to propose and obtain approval for a charter
+- [ ] Announce new WG on kubeflow-discuss@googlegroups.com
+
+### [GitHub]
+
+TODO: This section needs to be added for KF. In particular do we want to use [generator doc] which is what Kubernetes uses or is that
+too heavy weight?
+
+- [ ] Submit a PR that will add rows to [wgs.yaml] using the [generator doc]; this will create README files and OWNERS_ALIASES files for your new directory in `kubernetes/community`
+- You’ll need:
+  - WG Name
+  - Directory URL
+  - Mission Statement
+  - Chair Information
+  - Meeting Information
+  - Contact Methods
+  - Any WG Stakeholders
+  - Any Subproject Stakeholders
+- [ ] Add WG-related docs like charter.md, schedules, roadmaps, etc. to your new kubeflow/community/wg-foo directory once the above PR is merged.
+- [ ] File a [kubeflow/testing] Issue  and PR for a label; read about our [GitHub management] services
+
+### Communicate:
+
+WGs are responsible for setting up and administering the following communication channels
+- [ ] Mailing lists 
+      * foo@googlegroups.com
+      * foo-leads@googlegroups.com
+- [ ] Calendar for WG related meetings
+- [ ] Video conferencing (zoom, meet, etc...) 
+- [ ] Hosting for meeting notes, recordings etc...
+- [ ] Slack channels (if desired)
+
+### Engage:
+...as a chair/tech lead with other chairs/tech leads
+- [ ] Subscribe to the kubeflow-sig-leads@googlegroups.com group
+- [ ] Join the #chairs-and-techleads slack channel
+
+...with the community as part of [wg-governance.md]
+- [ ] Get on the Kubeflow community meeting agenda to provide WG updates
+- [ ] Create a shared calendar and schedule your weekly/biweekly/triweekly weeks [update meetings]
+- This calendar creation process will allow all of your leads to edit SIG/WG Meetings. This is important as we all change jobs, email addresses, and take breaks from the project. Shared calendars will also provide consistency with contributors looking for your subproject meetings, office hours, and anything else that the SIG/WGs contributors should know about.
```

Comment:
This doc is named "wg-lifecycle" and the mixed use of "SIG/WG" seems a bit confusing here.

Reply:
The doc is intended to cover the lifecycle of both SIGs and WGs. There were a couple of references that were incorrect that I updated.